modules/nf-neuro/registration/synthregistration/main.nf

```diff
@@ -40,7 +39,7 @@ process REGISTRATION_SYNTHREGISTRATION {
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
-        Freesurfer: 7.4
+        Freesurfer: 7.4.1
```

Comment:
Here wouldn't it make more sense to tag synthmorph version ? It somehow separated from freesurfer and its version is bound to change independently from it.

Reply:
There still are some inconsistencies with the official synthmorph:3 functionalities, which came more recently:

- joint registration
- including inverse transforms, which we can use for transform_trk

I suggest we write the synthmorph:3 version here, according to the docker version of synthmorph that is used, and then I open another PR that only focusses on fully upgrading the synthmorph module with the capabilities of the tool, which some of it will be useful.